bin/phoenixExport.js

```diff
@@ -66,10 +66,69 @@ function cleanupGeometry(node,
     }
 }
 
+//function to generate alternative colouring that the user can set within the config file
```

Comment:
Can you provide a brief explanation in English / bullet points on what this function does on a high level, i.e. don't describe what it does by following the code, but rather what its effects are in the end, and potentially also why things are done this way. 

IIUC: we are assigning colors to the `material`s that we find in the gltf (resp. to some property of that)?